### DISCUSSION TITLE
Facing:  error executing SQL statement [ERROR: role with OID 1126150 does not exist (28000)]

### DISCUSSION THREAD

**Discussion by sachin1kumar** [quarkusio/quarkus#39080] [Community]
**Complete stacktrace:**
org.hibernate.HibernateException - java.util.concurrent.CompletionException: org.hibernate.exception.GenericJDBCException: error executing SQL statement [ERROR: role with OID 1126150 does not exist (28000)] [select query]

Hiding the select query due to data privacy, when we run the query on OpenShift pod it works fine.

**Observation with this issue:**
Suppose currently we are facing this issue while using one of the endpoints of our microservice from Postman, making request multiple times but still same issue. Now, if we recreate the pod on Openshift then use the same endpoint we get the response and we will get the response till exact 1 hour and now if we use the same endpoint after one hour we will face the same issue again until we recreate the pod again on Openshift and this cycle repeats in order to make our endpoint works.

**Quarkus bom version:** 3.2.2.Final
**Quarkus vault version:** 'io.quarkiverse.vault:quarkus-vault:3.4.0'

**Comment by sberyozkin** [comment]
@sachin1kumar Please move it to https://github.com/quarkiverse/quarkus-vault

**Reply by sachin1kumar** [reply]
Moved: https://github.com/quarkiverse/quarkus-vault/discussions/241

**Reply by sberyozkin** [reply]
@sachin1kumar Thanks, let me close this one

**Reply by sberyozkin** [reply]
@sachin1kumar Would you be OK to delete it here ? I can't close it, as you probably did not use `Move` option, and it has no answer as such... Or may be mark my suggestion to move as an answer and then it can be closed, up to you, thanks

**Comment by sachin1kumar** [comment]
Closing this as we have moved this to quarkus vault.
